docker-shell-helper.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```

Comment:
This is a helper for me, so want it checked it. Other people are welcome to use, or ignore. There are instructions inline...  it just provide instructions to bring up a ruby env, and to be able to persist change in the image (if requried).